README.md

```diff
@@ -713,16 +713,20 @@ or show an account confirmed/welcome message in the case of `signup`, or direct
 
 One-Time-Password. Will deliver a magiclink or sms otp to the user depending on whether the request body contains an "email" or "phone" key.
 
+If `"create_user": true`, user will not be automatically signed up if the user doesn't exist.
+
 ```js
 {
   "phone": "12345678" // follows the E.164 format
+  "create_user": true
```

Comment:
I read the discussion but still a bit confused about the use case, sorry.

What is the default behavior if omitted?

And the dev still has the ability to prevent signups on these endpoints via some config if they want to enforce login only, right?

Reply:
if omitted, the default behaviour would be the current behaviour - which is that the user will be automatically signed up if they don't have an existing account, and logged in if they do.

Reply:
This flag isn't to restrict signups globally in their app. I think the use case here is for developers to have 2 separate flows - one for sign-up and one for sign-in when using magiclinks / otps.